cilium-health/launch/endpoint.go

```diff
@@ -224,7 +225,7 @@ func CleanupEndpoint() {
 //
 // CleanupEndpoint() must be called before calling LaunchAsEndpoint() to ensure
 // cleanup of prior cilium-health endpoint instances.
-func LaunchAsEndpoint(baseCtx context.Context, owner regeneration.Owner, n *node.Node, mtuConfig mtu.Configuration) (*Client, error) {
+func LaunchAsEndpoint(baseCtx context.Context, owner regeneration.Owner, n *node.Node, mtuConfig mtu.Configuration, lxcMap *lxcmap.LXCMap) (*Client, error) {
```

Comment:
Does this require to import `lxcmap` into this package or we can have an interface instead?

Reply:
I can hide this behind `Owner` for now and remove it as a member of `Endpoint`, which will remove this change entirely.